apis/medusa/v1alpha1/medusa_types.go

```diff
@@ -181,4 +181,11 @@ type MedusaClusterTemplate struct {
 	// Define the liveness probe settings to use for the Medusa containers.
 	// +optional
 	LivenessProbe *corev1.Probe `json:"livenessProbe,omitempty"`
+
+	// Should medusa backups be purged or not
+	// Defaults to true.
+	// +kubebuilder:default=true
+	// +optional
+
```

Comment:
Also, this empty line shouldn't probably be here. And the first line of comment should start with word `PurgeBackups ` so that the golang's automatic doc validator gets it.

Reply:
I think I've resolved this one.